generator/template/grpc_endpoint_conv.go

```diff
@@ -15,15 +15,11 @@ var (
 	defaultGolangTypes = []string{"string", "bool", "int", "uint", "byte", "int64", "uint64", "float64", "int32", "uint32", "float32"}
 )
 
-type GRPCConverterTemplate struct {
+type GRPCEndpointConverterTemplate struct {
 	PackagePath string
 }
 
-func utilPackagePath(path string) string {
-	return path + "/util"
-}
-
-func (t GRPCClientTemplate) converterPackagePath() string {
+func (t GRPCClientTemplate) grpcConverterPackagePath() string {
```

Comment:
This `func` must be declared in appropriate file near `GRPClientTemplate` struct